modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -156,6 +156,8 @@ The sources of seeds are called _seed providers_.
 
 The mechanism is pluggable, allowing new sources of seeds to be supported (see link:https://www.neo4j.com/docs/java-reference/current/extending-neo4j/project-setup/#extending-neo4j-plugin-seed-provider[Java Reference -> Implement custom seed providers] for more information).
 The product has built-in support for seed from a mounted file system (file), FTP server, HTTP/HTTPS server and Amazon S3.
+The S3 is supported by default, but the other providers require configuration of xref:reference/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[`dbms.databases.seed_from_uri_providers`].
```

Comment:
```suggestion
S3 is supported by default, but the other providers require configuration of xref:reference/configuration-settings.adoc#config_dbms.databases.seed_from_uri_providers[`dbms.databases.seed_from_uri_providers`].
```

Reply:
Thanks David, omitted _provider_ here. Better to omit _The_ as well.